st7789/st7789.go

```diff
@@ -373,35 +375,37 @@ func (d *Device) FillScreen(c color.RGBA) {
 	}
 }
 
+// Rotation returns the current rotation of the device.
+func (d *Device) Rotation() drivers.Rotation {
+	return d.rotation
+}
+
 // SetRotation changes the rotation of the device (clock-wise)
-func (d *Device) SetRotation(rotation Rotation) {
+func (d *Device) SetRotation(rotation Rotation) error {
 	madctl := uint8(0)
 	switch rotation % 4 {
-	case 0:
+	case drivers.Rotation0:
 		madctl = MADCTL_MX | MADCTL_MY
 		d.rowOffset = d.rowOffsetCfg
 		d.columnOffset = d.columnOffsetCfg
-		break
-	case 1:
+	case drivers.Rotation90:
 		madctl = MADCTL_MY | MADCTL_MV
 		d.rowOffset = d.columnOffsetCfg
 		d.columnOffset = d.rowOffsetCfg
-		break
-	case 2:
+	case drivers.Rotation180:
 		d.rowOffset = 0
 		d.columnOffset = 0
-		break
-	case 3:
+	case drivers.Rotation270:
 		madctl = MADCTL_MX | MADCTL_MV
 		d.rowOffset = 0
 		d.columnOffset = 0
-		break
```

Comment:
These `break` instructions aren't necessary in Go (it's the default behavior, unlike C/C++).